test/behavesLikeSqlFormatter.js

```diff
@@ -432,4 +432,25 @@ export default function behavesLikeSqlFormatter(language) {
         expect(format("'foo \\' JOIN bar'")).toBe("'foo \\' JOIN bar'\n");
         expect(format("`foo `` JOIN bar`")).toBe("`foo `` JOIN bar`\n");
     });
+
+    it("formats postgres cast operator", function() {
+        const result = sqlFormatter.format("column::int");
+        expect(result).toBe(
+            "column :: int\n"
+        );
```

Comment:
And we can move all of these new tests to 1 `it` block with description `formats PostgreSQL specific operators`.